node-core/components/chain_service.go

```diff
@@ -41,7 +40,6 @@ type ChainServiceInput struct {
 
 	ChainSpec             chain.Spec
 	Cfg                   *config.Config
-	EngineClient          *client.EngineClient
```

Comment:
not strictly relevant to this PR, but the EngineClient is not really used anymore here. So dropped